apps/new-tab/src/App.tsx

```
@@ -618,61 +822,138 @@
-                            );
-                            updateFilteredWidgets();
-                          }}
+                  Object.entries(widgetOrder()).map(
+                    ([item, value]: [any, string]) => {
+                      const widgetType = widgetOrder()[item];
+
+                      return (
+                        <div
+                          class={`${uuidv4()} slot group h-fit`}
+                          data-swapy-slot={item}
                         >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)